models/articles.js

```diff
@@ -2,12 +2,27 @@ const db = require("../db/connection");
 
 exports.selectArticleById = (articleId) => {
   return db
-    .query(`SELECT * FROM articles WHERE article_id = $1;`, [articleId])
-    .then((result) => {
-      if (result.rows.length === 0) {
+    .query(
+      `SELECT * FROM articles 
+    WHERE article_id = $1;`,
+      [articleId]
+    )
+    .then((articleInfo) => {
+      if (articleInfo.rows.length === 0) {
         return Promise.reject({ status: 404, msg: "article not found" });
+      } else {
+        return db
+          .query(
+            "SELECT COUNT(comment_id) FROM comments WHERE article_id = $1",
+            [articleId]
```

Comment:
You can combine these two queries into one big delicious query! Let me know if you get stuck doing it :)